SU2_CFD/src/solvers/CEulerSolver.cpp

```diff
@@ -4311,63 +4309,38 @@ void CEulerSolver::SetFarfield_AoA(CGeometry *geometry, CSolver **solver_contain
 
     /*--- Compute a new value for AoA on the fine mesh only (degrees)---*/
 
-    if (iMesh == MESH_0) AoA = AoA + AoA_inc;
-    else { AoA = config->GetAoA(); }
-
-    /*--- Only the fine mesh stores the updated values for AoA in config ---*/
-
     if (iMesh == MESH_0) {
+      AoA = AoA + AoA_inc;
       config->SetAoA(AoA);
     }
 
-    /*--- Update the freestream velocity vector at the farfield ---*/
-
-    for (iDim = 0; iDim < nDim; iDim++)
-      Vel_Infty[iDim] = GetVelocity_Inf(iDim);
-
-    /*--- Compute the magnitude of the free stream velocity ---*/
+    AoA *= PI_NUMBER/180.0;
 
-    Vel_Infty_Mag = 0;
-    for (iDim = 0; iDim < nDim; iDim++)
-      Vel_Infty_Mag += Vel_Infty[iDim]*Vel_Infty[iDim];
-    Vel_Infty_Mag = sqrt(Vel_Infty_Mag);
+    /*--- Update the freestream velocity vector at the farfield
+     * Compute the new freestream velocity with the updated AoA,
+     * "Velocity_Inf" is shared with config... ---*/
```

Comment:
```suggestion
     * "Velocity_Inf" is a class variable. ---*/
```

Reply:
LGTM to the little updates

Reply:
Not really because it is just a pointer to an array of CConfig, the data does not belong to the solver.
Horrible stuff but there we are...

Reply:
yes, I wanted to express that because I wondered for a brief second what 'shared with config' means. But yes, my suggestion is not correct indeed.
Out of curiosity: what way of sharing (or not sharing) you would prefer?